### PR TITLE
Cleanup std/no-std features

### PR DESCRIPTION
**Pull request by ryankurte**
fixes an issue with std/no_std, updates from `extern crate` to explicit includes.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -11,16 +11,17 @@ version = "0.10.0"
...
-std = []
 helpers = [ "structopt", "humantime", "std", "pcap-file", "libc", "byteorder", "rolling-stats" ]
 default = []
 
 [dependencies]
 embedded-hal = "1.0.0-alpha.5"
 nb = "1.0.0"
-log = "0.4.14"
+
+log = { version = "0.4.14", default_features = false }
```
Comment (henrikssn) [inline Cargo.toml]: Might be worth making this optional for embedded targets which are using `defmt` for logging

Comment (ryankurte) [reply Cargo.toml]: oh yeah, not a bad idea at all

File: Cargo.toml [hunk 2]
```diff
@@ -21,7 +21,8 @@ default = []
 embedded-hal = "1.0.0-alpha.5"
 nb = "1.0.0"
 
-log = { version = "0.4.14", default_features = false }
+log = { version = "0.4.14", optional = true, default_features = false }
+defmt = { version = "*", optional = true }
```
Comment (henrikssn) [inline Cargo.toml]: Please no star dependencies. We can use ">=0.2.0,<0.4" if we want to be backwards compatible during the 0.2/0.3 defmt migration.

**Comment by henrikssn** [comment]
You also need to rerun `cargo fmt`.

File: Cargo.toml (hunk 2 above)
Comment (ryankurte) [reply Cargo.toml]: oh good catch, it was just a placeholder that i forgot

**Comment by henrikssn** [comment]
LGTM!

**Pull request merged**